src/adhocracy_core/docs/authentication_api.rst

```diff
@@ -385,3 +385,67 @@ the user to be logged in from different devices at the same time. ::
     >>> resp_data = testapp.get('/meta_api/', headers=headers).json
     >>> 'resources' in resp_data.keys()
     True
+
+Alternative A: User Password reset without API extension (more backend friendly)
+--------------------------------------------------------------------------------
+
+A password reset link is send by creating a password reset object under the
+``/principals/passwordresets`` pool. The user is identified with the given
+email adress::
+
+    >>> prop = {'content_type': 'adhocracy_core.resources.principal.IPaswordReset',
+    ...         'data': {
+    ...              'adhocracy_core.sheets.principal.IPasswordReset': {
+    ...                  'email': 'anna@example.org'}}}
+    >>> resp_data = testapp.post_json(rest_url + "/principals/passwordresets", prop).json
+
+the url for the created password reset object is protected by obscurity::
+
+    >>> password_reset_path = resp_data["path"]
+    >>> password_reset_path
+    '.../principals/passwordresets/1318...
+
+    >>> resp = testapp.get(rest_url + "/principals/passwordresets")
+    >>> resp.status_code
+    404
+
+On Success the backend sends an email with the link to reset the password to
+the user. The link contains the path to the created passwordreset object and
+needs to be handled by the frontend::
+
+    http://frontend_url/passwordreset/?path=/principals/passwordresets/1318...
+
+If the user clicks on this link, the frontend has to send a put request with the
+new password::
+
+  >>> prop = {'data': {'adhocracy_core.sheets.principal.IPasswordAuthentication': {
+    ...                     'password': 'new_password'}}}
+    >>> resp = testapp.put_json(password_reset_path, prop)
+    >>> resp.status_code
+    200
+
+Alternative B: User Password reset with API extension (more thentos friendly) 
+---------------------------------------------------------------------------------
+
+The frontend sends an email to the create password reset end point
+
+    >>> prop = {'email': 'anna@example.org'}
+    >>> resp = testapp.post_json(rest_url + "/create_password_reset", prop)
+    >>> resp.status_code
+    200
+
+On Success the backend sends an email with the link to reset the password to
+the user. The link contains the path to identify the password reset request::
+
+    http://frontend_url/passwordreset/?path=1318...
+
+If the user clicks on this link, the frontend has to send a post request with the
+new password to the reset password end point::
+
+    >>> newest_reset_path = getfixture('newest_reset_path')
+    >>> prop = {'path': newest_reset_path,
+                'password': new_password}
+    >>> resp = testapp.post_json('/reset_password', prop)
+    >>> resp.status_code
+    200
+
```

Comment:
Regarding  version A  or B,  I would prefere the solution that is les work for backend/frontend.